app/policies/portal/offering_policy.rb

```diff
@@ -1,2 +1,17 @@
 class Portal::OfferingPolicy < ApplicationPolicy
+
+  def show?
+    class_teacher_or_admin?
+  end
```

Comment:
It would be great if this was expanded to allow project admins too. But it is complex, so I think we'll need to wait until we have some time to focus on that later.